docs/core/config/namespaces.md

```diff
@@ -42,6 +42,30 @@ All visible namespaces must have the same separator.
 
 Inside each namespace there is a list of folders, which form a sub-hierarchy.
 
+::: info
+Enable default INBOX namespace and English language mailbox names with mailbox_special_use flags with [[setting,@mailbox_defaults,english]]
```

Comment:
missing setting notation from mailbox_special_use

Reply:
Has a separate MR been created for this yet?